.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
We could also use [Discussions](https://github.com/Amsterdam/design-system/discussions) for this.